filebeat/docs/fields.asciidoc

```diff
@@ -49927,6 +49927,49 @@ type: boolean
 
 
 
+
+*`elasticsearch.deprecation.data_stream.dataset`*::
++
+--
+type: keyword
+
+--
+
+*`elasticsearch.deprecation.data_stream.namespace`*::
++
+--
+type: keyword
+
+--
+
+*`elasticsearch.deprecation.data_stream.type`*::
++
+--
+type: keyword
+
+--
+
+*`elasticsearch.deprecation.event.code`*::
++
+--
+type: keyword
+
+--
+
+*`elasticsearch.deprecation.event.dataset`*::
```

Comment:
I don't think we should be adding these to the documentation. Maybe @pgomulka can confirm but fields that don't have the `elasticsearch.` prefix appear to be all ecs fields (for example this one is https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-dataset)

We can lift the field up to the root of the document to avoid having to document them here.

Reply:
For example I did this for `trace.id` from the slow logs:

```
- dot_expander:
    field: trace.id
    path: elasticsearch.slowlog
- rename:
    field: elasticsearch.slowlog.trace.id
    target_field: trace.id
    ignore_missing: true
```

Reply:
> I don't think we should be adding these to the documentation. Maybe @pgomulka can confirm but fields that don't have the elasticsearch. prefix appear to be all ecs fields (for example this one is https://www.elastic.co/guide/en/ecs/current/ecs-event.html#field-event-dataset)
We can lift the field up to the root of the document to avoid having to document them here.

yes, as mentioned in other comments. Logs emitted by ES8 are ECS and don't need to be renamed. Could be just attached to the root of the document.

Reply:
actually @ruflin 's idea might work even better. Just use add_to_root+dot_expander and you can avoid a lot of the remapping I think